src/olympia/applications/templates/applications/appversions.html

```diff
@@ -15,6 +15,10 @@
 allowed for these applications.
 {% endtrans %}</p>
 
+<p >{% trans version_url=_('https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/applications') %}
```

Comment:
Also, do we prefer having the developers translate the locale each time rather than linking without the locale and letting the other site figure it out? Seems just omitting the locale is easier.